runtime/tests/port/j9vmemTest.c

```diff
@@ -2238,14 +2260,43 @@ j9vmem_testFindValidPageSize_impl(struct J9PortLibrary *portLibrary, char *testN
 		expectedPageFlags = defaultLargePageFlags;
 		expectedIsSizeSupported = FALSE;
 	} else {
-		expectedPageSize = FOUR_KB;
+		expectedPageSize = defaultPageSize;
 		expectedPageFlags = J9PORT_VMEM_PAGE_FLAG_NOT_USED;
 		expectedIsSizeSupported = FALSE;
 	}
 
 	verifyFindValidPageSizeOutput(portLibrary, testName,
-								expectedPageSize, expectedPageFlags, expectedIsSizeSupported,
-								requestedPageSize, requestedPageFlags, isSizeSupported);
+	                              expectedPageSize, expectedPageFlags, expectedIsSizeSupported,
+	                              requestedPageSize, requestedPageFlags, isSizeSupported);
+
+	/* 2M is only possible for Linux */
+#if defined(LINUXPPC)
+	outputComment(PORTLIB, "\nCase %d: -Xlp:objectheap:pagesize=2M\n", caseIndex++);
+
+	requestedPageSize = 2 * ONE_MB;
+	requestedPageFlags = J9PORT_VMEM_PAGE_FLAG_NOT_USED;
+	PRINT_FIND_VALID_PAGE_SIZE_INPUT(MODE_NOT_USED, requestedPageSize, requestedPageFlags);
+
+	j9vmem_find_valid_page_size(MODE_NOT_USED, &requestedPageSize, &requestedPageFlags, &isSizeSupported);
+
+	if (TRUE == twoMBPageSize) {
+		expectedPageSize = 2 * ONE_MB;
+		expectedPageFlags = J9PORT_VMEM_PAGE_FLAG_NOT_USED;
+		expectedIsSizeSupported = TRUE;
+	} else if (0 != defaultLargePageSize) {
+		expectedPageSize = defaultLargePageSize;
+		expectedPageFlags = defaultLargePageFlags;
+		expectedIsSizeSupported = FALSE;
+	} else {
+		expectedPageSize = defaultPageSize;
+		expectedPageFlags = J9PORT_VMEM_PAGE_FLAG_NOT_USED;
+		expectedIsSizeSupported = FALSE;
+	}
+
+	verifyFindValidPageSizeOutput(portLibrary, testName,
+	                              expectedPageSize, expectedPageFlags, expectedIsSizeSupported,
+	                              requestedPageSize, requestedPageFlags, isSizeSupported);
```

Comment:
Indentation.

Reply:
See https://github.com/eclipse-omr/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line. (Unfortunately, the continued lines are shown indented 8 spaces instead of 2 tabs, in the correct examples. but the text above them says "at least 1 tab"; as tabs are treated like 4 spaces, 8 spaces is 2 tabs.)